compoments/report-repair/address.js

```diff
@@ -48,6 +48,9 @@ const Address = ({handleChange, values}) => {
   }
 
   return <div className="govuk-grid-row" data-cy="address">
+    <header>
+      <title>Select an address- Housing repairs</title>
```

Comment:
The first part of the title text matches the title (in the [H1 tag](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/3/files#diff-3ded2902c4403e2811a2b3bfca88eda2eeba6fa7291295268dfa31d646395eebR55)) displayed on the page. 

Extract the content to a variable and use in both locations to reduce duplication and increase maintainability (DRY principle).

This is relevant to almost all of the changed components - some already had a `title`, such as [communal.js](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/3/files#diff-795c80141a8fd3e083ab989770a1cd60c2467d4eb641e53a4ab18f873cc60437R9).